src/main/kotlin/com/atlassian/performance/tools/report/Util.kt

```diff
@@ -0,0 +1,18 @@
+package com.atlassian.performance.tools.report
+
+import com.atlassian.performance.tools.report.api.result.EdibleResult
+
+internal class Util {
```

Comment:
`Util` is a bad name, because it's the vaguest of them all. It has no palpable responsibility. It's a bag for random code. It will attract more random code into it, degrading the overall design.

`toDoubleArray` is a bridge between `EdibleResult` and `NonparametricFacade`. We need this transformation, because the underlying lib: commons-math operates on `Double`s. That's fine. But we already had that problem and we solved it via `DurationData.stats.values` obtainable from `ActionMetricsReader`. It has the added bonus of not forgetting to filter out failed actions and not forgetting about the time unit.